src/features/projects/styles/ProjectDetails.module.scss

```diff
@@ -320,7 +320,9 @@
     min-height: calc(100vh - 100px);
     background-color: transparent;
   }
-
+  .reports_container{
+    width: 360px;
+  }
```

Comment:
This still does not work for me as this container is still the same size as the `projectContainer` it is part of despite some paddings. Try removing this. It will lead to a different design in the list, but at least not cut off any text.

1. http://localhost:3000/jaguarsiembra?site=Bunkwimake
<img width="384" alt="Bildschirm­foto 2022-11-26 um 13 18 22" src="https://user-images.githubusercontent.com/1532418/204088564-1954568c-a489-41f9-b344-0d0cb762ae31.png">

2. List view:
<img width="850" alt="Bildschirm­foto 2022-11-26 um 13 18 36" src="https://user-images.githubusercontent.com/1532418/204088580-1f8ab867-0ffe-4320-92bb-9d246ab3f7f6.png">